Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 resolver = "2"
 members = [
+	"bindings",
```

Comment:
I think I would already have a `core` subfolder, because at some point we'll have the bindings in there